lib/dradis/plugins/veracode/field_processor.rb

```diff
@@ -4,14 +4,15 @@ module Veracode
       class FieldProcessor < Dradis::Plugins::Upload::FieldProcessor
         def post_initialize(args = {})
           @record =
-            if (data.is_a?(::Veracode::Flaw) || data.is_a?(::Veracode::Evidence))
+            if data.is_a?(::Veracode::Flaw) || data.is_a?(::Veracode::Evidence) || data.is_a?(::Veracode::Vulnerability)
               data
-
             # Note: The evidence and flaw samples are the same but they need to
             # be differentiated in the plugins manager preview. In that case,
             # we're adding a "dradis_type" attribute in the evidence.sample file
-            elsif (data['dradis_type'] == 'evidence')
+            elsif data['dradis_type'] == 'evidence'
               ::Veracode::Evidence.new(data.at_xpath('./staticflaws/flaw'))
+            elsif data.name == 'component'
+              ::Veracode::Vulnerability.new(data.at_xpath('.//vulnerability'))
```

Comment:
@caitmich is ".//" really needed here?

Reply:
Fixed this to search a specific path: `./vulnerabilities/vulnerability`